dot/network/service.go

```diff
@@ -316,14 +317,11 @@ func (s *Service) sentBlockIntervalTelemetry() {
 			continue
 		}
 
-		telemetry.GetInstance().SendBlockIntervalData(&telemetry.BlockIntervalData{
-			BestHash:           best.Hash(),
-			BestHeight:         best.Number,
-			FinalizedHash:      finalized.Hash(),
-			FinalizedHeight:    finalized.Number,
-			TXCount:            0, // todo (ed) determine where to get tx count
-			UsedStateCacheSize: 0, // todo (ed) determine where to get used_state_cache_size
-		})
+		telemetry.GetInstance().SendMessage(telemetry.NewTelemetryMessage(telemetry.NewKeyValue("best", best.Hash().String()),
+			telemetry.NewKeyValue("finalized_hash", finalized.Hash().String()), telemetry.NewKeyValue("finalized_height", finalized.Number), //nolint
```

Comment:
the block here is the best hash, not the best finalized hash

Reply:
@noot I'm confused, in there a difference between finalized hash and best finalized hash?  How do I obtain finalized hash?

Reply:
```
telemetry.GetInstance().SendMessage(telemetry.NewTelemetryMessage(
	telemetry.NewKeyValue("best", best.Hash().String()),
	telemetry.NewKeyValue("finalized_hash", finalized.Hash().String()),
	telemetry.NewKeyValue("finalized_height", finalized.Number), //nolint
	telemetry.NewKeyValue("height", best.Number), telemetry.NewKeyValue("msg", "system.interval"),
	telemetry.NewKeyValue("txcount", 0),                // todo (ed) determine where to get tx count
	telemetry.NewKeyValue("used_state_cache_size", 0)), // todo (ed) determine where to get used_state_cache_size
)
```

Also, have a single `NewKeyValue` per line.

Reply:
yes, nevermind, I read this wrong, it's actually fine